### DISCUSSION TITLE
Is there any way to define many to many relationship?

### DISCUSSION THREAD

**Discussion by dgvai** [supabase/supabase#1336] [Questions]
What I have only found is defining foreign key relations to another table. I can not managed to fine the many to many relationship definition. So do I manually need to create a third table with `table_1_id` and `table_2_id` column define their foreign key and go after so on?

**Comment by inian** [comment]
Yup, that would be the way @dgvai. 
Haven't tried this but I think after you do that, you can [query it with postgrest resource embedding](https://supabase.io/docs/reference/javascript/select#query-foreign-tables) without specifying the intermediate join table.
